charts/newrelic-infra-operator/README.md

```diff
@@ -36,7 +36,7 @@ This is typically used in environments where DaemonSets are not available, such
 | `priorityClassName`                                                             | Scheduling priority of the pod.                                                                                                                                                                                                                                                                                                             | `nil`                                                                                                |
 | `nodeSelector`                                                                  | Node label to use for scheduling.                                                                                                                                                                                                                                                                                                           | `{}`                                                                                                 |
 | `timeoutSeconds`                                                                | Seconds to wait for a webhook to respond. The timeout value must be between 1 and 30 seconds.                                                                                                                                                                                                                                               | `30`                                                                                                 |
-| `tolerations`                                                                   | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                                                                                                                                | `[]`                                                                                                 |
+| `tolerations`                                                                   | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                                                                                                                                | `See Tolerations below`                                                                              |
```

Comment:
```suggestion
| `tolerations`                                                                   | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                                                                                                                                | Tolerations for the operator deployment and jobs. See `Tolerations` below.                                                                              |
```

Reply:
Modified the file with the suggestions

Reply:
Lint might fail again, I need to disable few stuff on my IDE so files don't get auto formated on save.

If you have by anychance a `.editorconfig` files with your standards would be great.

Thanks,